docs/configuration/configuration.md

```diff
@@ -1350,9 +1350,13 @@ See below for the configuration options for Kubernetes discovery:
 # One of endpoints, service, pod, node, or ingress.
 role: <string>
 
+# Optional kube_config will use kube_config file to connect to API server
+# `kube_config` the filename of kube_config,
+# Note that api_server and kube_config are mutually exclusive.
+[ kube_config: <filename> ]
```

Comment:
kubeconfig is never spelled in 2 words, what about using kubeconfig_file as parameter name? It would be more consistent with other fields.

Reply:
ok, I rename it .